homeassistant/helpers/dispatcher.py

```diff
@@ -151,11 +151,11 @@ def _format_err[*_Ts](
     *args: Any,
 ) -> str:
     """Format error message."""
-    return "Exception in {} when dispatching '{}': {}".format(
+
+    return (
         # Functions wrapped in partial do not have a __name__
-        getattr(target, "__name__", None) or str(target),
-        signal,
-        args,
+        f"Exception in {getattr(target, "__name__", None) or target} "
```

Comment:
```suggestion
        f"Exception in {getattr(target, '__name__', None) or target} "
```

Reply:
This is already supported in Python 3.12: https://docs.python.org/3/whatsnew/3.12.html#pep-701-syntactic-formalization-of-f-strings

Reply:
Ok.